webpack.config.js

```diff
@@ -41,6 +41,63 @@ const gutenbergPackages = Object.keys( dependencies )
 	)
 	.map( ( packageName ) => packageName.replace( WORDPRESS_NAMESPACE, '' ) );
 
+const transformBlockContent = ( content ) => {
+	content = content.toString();
+
+	// Within content, search for any function definitions. For
+	// each, replace every other reference to it in the file.
+	return (
+		content
+			.match( /^function [^\(]+/gm )
+			.reduce( ( result, functionName ) => {
+				// Trim leading "function " prefix from match.
+				functionName = functionName.slice( 9 );
+
+				// Prepend the Gutenberg prefix, substituting any
+				// other core prefix (e.g. "wp_").
+				return result.replace(
+					new RegExp( functionName, 'g' ),
+					( match ) => 'gutenberg_' + match.replace( /^wp_/, '' )
+				);
+			}, content )
+			// The core blocks override procedure takes place in
+			// the init action default priority to ensure that core
+			// blocks would have been registered already. Since the
+			// blocks implementations occur at the default priority
+			// and due to WordPress hooks behavior not considering
+			// mutations to the same priority during another's
+			// callback, the Gutenberg build blocks are modified
+			// to occur at a later priority.
+			.replace(
+				/(add_action\(\s*'init',\s*'gutenberg_register_block_[^']+'(?!,))/,
+				'$1, 20'
+			)
+	);
+};
+
+const blocksDirectoryMapping = toPairs( {
```

Comment:
Nitpick: Personally, I'm not a fan of over-using lodash in simple cases like this one, prefer just a simple `Object.entries()` call 😅